notebooks/official/matching_engine/README.md

```diff
@@ -11,8 +18,27 @@ The steps performed include:
 * Perform online query
 * Compute recall
 
+    <details>
```

Comment:
Is this a markdown tag?

Reply:
In github markdown this creates an expandable/collapsible element:

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections